### PR TITLE
loading inline

### PR DESCRIPTION
**Pull request by fraincs**
Issue:

https://github.com/gsoft-inc/sg-orbit/issues/323

## How to test

- Is this testable with Jest or Chromatic screenshots?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.

### CODE REVIEW

File: packages/react-components/src/select/stories/inline.stories.mdx [hunk 1]
```diff
@@ -104,6 +104,23 @@ An inlined select can be embedded in a block.
     </Story>
 </Preview>
 
+### Loading
+
+An inlined select can show that it is currently loading data.
+
+<Preview>
+    <Story name="loading">
+        <div><Select
```
Comment (patricklafrance) [inline packages/react-components/src/select/stories/inline.stories.mdx:113]: You don't need the <div></div>

**Comment by patricklafrance** [comment]
As you said, it looks weird because it's not centered for all font-size.

I am not sure what to think of this, we need it but at the same time it doesn't look very professional to release something that is not centered.

File: packages/react-components/src/select/stories/inline.stories.mdx (hunk 1 above)
Comment (fraincs) [reply packages/react-components/src/select/stories/inline.stories.mdx:113]: the what

Comment (patricklafrance) [reply packages/react-components/src/select/stories/inline.stories.mdx:113]: the div

**Pull request merged**